api/pkg/store/mongo/store.go

```diff
@@ -23,37 +23,8 @@ func NewStore(db *mongo.Database) *Store {
 	return &Store{db: db}
 }
 
-func (s *Store) ListDevices(ctx context.Context, perPage, page int, filters []models.Filter) ([]models.Device, int, error) {
-	skip := perPage * (page - 1)
-	var queryFilter []bson.M
-	for _, filter := range filters {
-		if filter.Type == "property" && filter.Params.Operator == "like" {
-			queryFilter = append(queryFilter, bson.M{
-				filter.Params.Name: bson.M{
-					"$regex":   filter.Params.Value,
-					"$options": "i",
-				},
-			})
-		} else if filter.Type == "property" && filter.Params.Operator == "eq" {
-			queryFilter = append(queryFilter, bson.M{
-				filter.Params.Name: bson.M{
-					"$eq": filter.Params.Value,
-				},
-			})
-		} else if filter.Type == "property" && filter.Params.Operator == "bool" {
-			operator, err := strconv.ParseBool(filter.Params.Value)
-			if err != nil {
-				return nil, 0, err
-			}
-			queryFilter = append(queryFilter, bson.M{
-				filter.Params.Name: bson.M{
-					"$eq": operator,
-				},
-			})
-		}
-	}
-
-	query := []bson.M{
+func queryDevices() []bson.M {
```

Comment:
Functions in a file should be grouped by receiver and non-receiver (and unexported) functions should appear in the end of file.

Reply:
I made this change, please take a look if it looks good now.